core/controllers/topic_viewer.py

```diff
@@ -78,8 +81,21 @@ def get(self, topic_name):
         subtopics = topic.get_all_subtopics()
 
         assigned_skill_ids = topic.get_all_skill_ids()
-        skill_descriptions = skill_services.get_skill_descriptions_by_ids(
-            topic.id, assigned_skill_ids)
+        skill_descriptions, deleted_skill_ids = (
+            skill_services.get_descriptions_of_skills(
+                assigned_skill_ids))
+
+        if deleted_skill_ids:
+            deleted_skills_string = ', '.join(deleted_skill_ids)
+            logging.error(
+                'The deleted skills: %s are still present in topic with id %s'
+                % (deleted_skills_string, topic.id)
+            )
+            if feconf.CAN_SEND_EMAILS:
```

Comment:
Once again, we should batch these emails / or send them to some google group that people can subscribe to.